code/notificationbuilder/src/main/res/values/styles.xml

```diff
@@ -13,14 +13,6 @@
 
 -->
 <resources>
-    <style name="Messaging.Style.Transparent" parent="@android:style/Theme.Translucent.NoTitleBar">
```

Comment:
Moved this to the test app and caller extension as its used by the tracker activity and should not be present in the UI library